src/gui_common/dialogs/PermanentlyDismissableDialog.tscn

```diff
@@ -0,0 +1,84 @@
+[gd_scene load_steps=5 format=2]
+
+[ext_resource path="res://src/gui_common/dialogs/CustomConfirmationDialog.tscn" type="PackedScene" id=1]
+[ext_resource path="res://src/gui_common/CustomCheckBox.tscn" type="PackedScene" id=2]
+[ext_resource path="res://src/gui_common/fonts/Lato-Regular-Small.tres" type="DynamicFont" id=4]
+[ext_resource path="res://src/gui_common/dialogs/PermanentlyDismissableDialog.cs" type="Script" id=5]
+
+[node name="PermanentlyDismissableDialog" instance=ExtResource( 1 )]
+script = ExtResource( 5 )
+NoticeType = 0
+
+[node name="VBoxContainer" parent="." index="0"]
+margin_right = 193.0
+margin_bottom = 116.0
+
+[node name="Spacer5" type="Control" parent="VBoxContainer" index="0"]
+margin_right = 178.0
+margin_bottom = 15.0
+rect_min_size = Vector2( 0, 15 )
+
+[node name="Spacer" parent="VBoxContainer" index="1"]
+margin_top = 15.0
+margin_right = 178.0
+margin_bottom = 15.0
+
+[node name="Label" parent="VBoxContainer" index="2"]
+margin_top = 15.0
+margin_right = 178.0
+margin_bottom = 37.0
+
+[node name="Spacer2" parent="VBoxContainer" index="3"]
+margin_top = 37.0
+margin_right = 178.0
+margin_bottom = 37.0
+
+[node name="Spacer3" parent="VBoxContainer" index="4"]
+margin_top = 37.0
+margin_right = 178.0
+margin_bottom = 52.0
+
+[node name="CheckBox" parent="VBoxContainer" index="5" instance=ExtResource( 2 )]
+margin_top = 52.0
+margin_right = 178.0
+margin_bottom = 69.0
+custom_fonts/font = ExtResource( 4 )
+pressed = true
+text = "DO_NOT_SHOW_AGAIN"
```

Comment:
Removing the don't show again is not very good, the reason that text exists is that some dismissible popups are not warnings. So the text that "don't warn again" does not fit all usages.

Reply:
Though, it's a bit amusing how the translation key itself does not reflect the actual text 😁

Reply:
That's the power of the translation keys, we can adjust the text to taste whenever we want without having to modify the code.